app/script/util/util.coffee

```diff
@@ -653,3 +654,35 @@ z.util.bucket_values = (value, bucket_limits) ->
 
   last_limit = bucket_limits[bucket_limits.length - 1]
   return "#{last_limit+1}-"
+
+
+# http://stackoverflow.com/a/1064139
+z.util.insert_at_caret = (txtarea, text) ->
```

Comment:
Done.. 

By they way.. this new review thingy is not removing the outdated comments :(